index.js

```diff
@@ -20,7 +20,7 @@ const exec = (cmd, args) => {
 	}
 };
 
-module.exports = input => {
+module.exports = (input, opts) => {
```

Comment:
Might be somewhat moot but I'd prefer to also have `opts = opts || {}`. You never know if someone uses the API directly in another project. Making sure `opts` is not `undefined` seems best practice in any case.